src/releases.js

```diff
@@ -10,7 +10,7 @@ const parseReleases = async (tags, options, onParsed) => {
     const merges = commits.filter(commit => commit.merge).map(commit => commit.merge)
     const fixes = commits.filter(commit => commit.fixes).map(commit => ({ fixes: commit.fixes, commit }))
 
-    for (const plugin of options.plugins) {
+    for (const plugin of Array.from(options.plugins || [])) {
```

Comment:
why Array.from? what will it be if it's truthy but not an array?

Reply:
I suggested Array.from because it's the shortest way to check if `plugins` is iterable. any other suggestion?

In that case the test might pass due to the conditions inside the loop.

Reply:
`[].concat(options.plugins || [])` will be much faster, but also it's *fine* if options.plugins throws here when passed a truthy non-iterable value.